ruby/vampires.rb

```diff
@@ -1,3 +1,4 @@
+# How many people to survey
 puts "How many people are we testing?"

```

Comment:
For vamp conditions checks

Lets break the evaluation conditions out of the questions. It is always good to finish one job before starting another. Collect all of the information you need about your employee and then move to eval separately.

Consider the optimum condition when running your code.  What can we do to run only necessary processes?  We know that if the employee has Drac or Fang in their name then they are definitely a vampire.  If that is the case, do we need to run any of the other checks?

Consider:

```
if name_matches_vampire_params
  "Definitely a vampire"
else
  if #other checks
    "Something"
  elsif #more conditions...

  else
  end
end
```